static/app/components/list/index.tsx

```diff
@@ -3,10 +3,13 @@ import styled from '@emotion/styled';
 
 import {space} from 'sentry/styles/space';
 
+import {ListItemProps} from './listItem';
 import {getListSymbolStyle, listSymbol} from './utils';
 
+type ListItemChild = React.ReactElement<ListItemProps> | undefined | false;
```

Comment:
why `false`? I don't think we want the children to be of type boolean

Reply:
This is what happens when you do {myVar && <Whatever />